paasta_tools/cli/cmds/local_run.py

```diff
@@ -735,7 +761,9 @@ def _cleanup_container(docker_client, container_id):
         )
 
 
-def get_local_run_environment_vars(instance_config, port0, framework):
+def get_local_run_environment_vars(
+    instance_config, port0, framework, container_hostname
```

Comment:
for extra credit, i'd be nice to type this - but non-blocking :p

Reply:
done!